### PR TITLE
WIP (don't merge): Update testing.md

### PR DESCRIPTION
**Pull request by tunniclm**
A few corrections
Extra examples
Link to a working example test project (I have one, but it is not uploaded anywhere yet)

### CODE REVIEW

**Comment by ianpartridge** [comment]
@tunniclm any update on this?

**Comment by tunniclm** [comment]
@ianpartridge The outstanding work here (outside of reviewing the text) is to publish the example test project.

File: en/starter/testing.md [hunk 1]
```diff
@@ -16,51 +16,40 @@ This article will walk you through setting up tests in accordance of the XCTest
 
 # Setting up the Tests directory
 1. In the root level of your project, create a folder called `Tests`.
-2. Inside the `Tests` directory you'll need to create a folder for holding your test suite module. It is recommended that it is the same name as your App module.
+2. Inside the `Tests` directory you'll need to create a folder for holding your test suite module. It is recommended that it is the same name as your App module, suffixed with `Tests` (for example: `AppTests`).
 
 # Creating Test Cases
-1. Create a new Swift file in the test suite folder.
+1. Create a new Swift file in the test suite folder (for example: `AppTests/FooTests.swift`)
 2. When setting up this file you'll want to add `@testable import <module-name>` for each of the modules you want to test. (Note the `@testable` keyword allows you to call internal methods in your tests)
```
Comment (jacobvanorder) [inline en/starter/testing.md]: There should be a step between 2. and 3. that says to include `import XCTest` as well.

**Comment by ianpartridge** [comment]
@tunniclm Has the example test project been published now? Can we fix up this PR and get it merged?

**Comment by tunniclm** [comment]
@ianpartridge I made a new commit to fix up the content, but it looks like there were some formatting changes. Perhaps we can get together and pick out the content changes from the formatting changes and get this pr into a merge-able state.

**Comment by tunniclm** [comment]
@ianpartridge Actually, I had a crack at this in a new pull request (#32) based on `master`. Closing this PR in preference to the new one.

**Pull request closed without merging**